backend/src/routes/projects/projects.ts

```diff
@@ -174,6 +179,11 @@ projectRouter.get('/status/:id', async (req: Request, res: Response) => {
 
 projectRouter.post('/', async (req: Request, res: Response) => {
     try {
+        for (let i: number = 0; i < req.body.elementsIds.length; i++) {
+            const result: any = await dataBase.getRepository(Inventory).createQueryBuilder('inventory').where('inventory.id = :id', { id: req.body.elementsIds[i] }).setParameter('id', String(req.body.elementsIds[i])).getOne();
+            result.quantity = result.quantity - 1;
```

Comment:
remove quantity received from front not 1

Reply:
How the body would look like then? Because elementIds is an array of ids
For example if an element was used more than one it would appear several times in the array